libs/cluster/Server/Replication/PrimaryOps/AofSyncTaskInfo.cs

```diff
@@ -55,7 +55,7 @@ public void Dispose()
             cts?.Dispose();
         }
 
-        public unsafe void Consume(byte* payloadPtr, int payloadLength, long currentAddress, long nextAddress)
+        public unsafe void Consume(byte* payloadPtr, int payloadLength, long currentAddress, long nextAddress, bool isProtected)
```

Comment:
isProtected is no longer necessary here

Reply:
Noting that this is the Primary AOF iteration. It does not use isProtected because it does not Acquire the epoch when consuming. The isProtected is used for the Consume part of the replica replay task.